test/unit/server.js

```diff
@@ -6,31 +6,36 @@ var assert = chai.assert;
 var sinon = require('sinon');
 
 var Promise = require('bluebird');
-var Worker = require('../../lib/worker');
 var hermes = require('runnable-hermes');
 var noop = require('101/noop');
+
 var ponos = require('../../');
+var Worker = require('../../lib/worker');
+var log = require('../../lib/logger');
```

Comment:
let's rename this to something like `ponosDefaultLogger` or something... I saw `log` down below in an `assert.equal` and was super confused